src/mame/audio/vicdual-97271p.h

```diff
@@ -0,0 +1,45 @@
+// license:BSD-3-Clause
+// copyright-holders:Ariane Fugmann
+#pragma once
+
+#ifndef __S97271P_H__
+#define __S97271P_H__
```

Comment:
Please don't use __names - they are reserved for the compiler/runtime according to the standard.  Make it match the path/filename: `MAME_INCLUDES_VICDUAL_97271P_H`